test/util/cmr.ts

```diff
@@ -11,5 +11,10 @@ describe('util/cmr', function () {
       const variables = await getVariablesByIds(ids, '');
       expect(variables.length).to.eql(1);
     });
+
+    it('response contains related URLs when the CMR variable has them', async function () {
```

Comment:
You can use `xit` to tell it not to run. Normally we would include a ticket number in the comment to explain why it is not running.

Reply:
Updated.